cmd/cloud-controller-manager/app/options/options.go

```diff
@@ -129,21 +122,21 @@ func (o *CloudControllerManagerOptions) Flags() apiserverflag.NamedFlagSets {
 	fss := apiserverflag.NamedFlagSets{}
 	o.Generic.AddFlags(&fss, []string{}, []string{})
 	// TODO: Implement the --controllers flag fully for the ccm
-	fss.FlagSet("generic").MarkHidden("controllers")
-	o.KubeCloudShared.AddFlags(fss.FlagSet("generic"))
-	o.ServiceController.AddFlags(fss.FlagSet("service controller"))
+	fss.FlagSet(CloudControllerManagerGeneric).MarkHidden(CloudControllerManagerHiddenMark)
+	o.KubeCloudShared.AddFlags(fss.FlagSet(CloudControllerManagerGeneric))
+	o.ServiceController.AddFlags(fss.FlagSet(CloudControllerManagerServiceController))
 
-	o.SecureServing.AddFlags(fss.FlagSet("secure serving"))
-	o.InsecureServing.AddUnqualifiedFlags(fss.FlagSet("insecure serving"))
-	o.Authentication.AddFlags(fss.FlagSet("authentication"))
-	o.Authorization.AddFlags(fss.FlagSet("authorization"))
+	o.SecureServing.AddFlags(fss.FlagSet(CloudControllerManagerSecureServing))
+	o.InsecureServing.AddUnqualifiedFlags(fss.FlagSet(CloudControllerManagerInsecureServing))
+	o.Authentication.AddFlags(fss.FlagSet(CloudControllerManagerAuthentication))
+	o.Authorization.AddFlags(fss.FlagSet(CloudControllerManagerAuthorization))
 
-	fs := fss.FlagSet("misc")
-	fs.StringVar(&o.Master, "master", o.Master, "The address of the Kubernetes API server (overrides any value in kubeconfig).")
-	fs.StringVar(&o.Kubeconfig, "kubeconfig", o.Kubeconfig, "Path to kubeconfig file with authorization and master location information.")
-	fs.DurationVar(&o.NodeStatusUpdateFrequency.Duration, "node-status-update-frequency", o.NodeStatusUpdateFrequency.Duration, "Specifies how often the controller updates nodes' status.")
+	fs := fss.FlagSet(CloudControllerManagerMisc)
```

Comment:
Line 133 to line 141, those section flag( such as `generic`、`service controller` and so on)also using by KCM, i would like hold it in `cmd/controller-manager` directory